var/spack/repos/builtin/packages/crtm/package.py

```diff
@@ -13,8 +13,17 @@ class Crtm(CMakePackage):
     scattering, and a solver for a radiative transfer."""
 
     homepage = "https://www.jcsda.org/jcsda-project-community-radiative-transfer-model"
-    url      = "https://github.com/NOAA-EMC/EMC_crtm/archive/refs/tags/v2.3.0.tar.gz"
+    git = 'https://github.com/JCSDA/crtm.git'
 
     maintainers = ['t-brown', 'edwardhartnett', 'kgerheiser', 'Hang-Lei-NOAA']
 
-    version('2.3.0', sha256='3e2c87ae5498c33dd98f9ede5c39e33ee7f298c7317b12adeb552e3a572700ce')
+    depends_on('netcdf-fortran', when='@2.4.0:')
+
+    # ecbuild release v2.4.0 is broken
```

Comment:
What is the problem?

Reply:
The 2.4.0 release is missing the version in the CMake project declaration and ecbuild doesn't like that. This has been fixed in the later JEDI releases.

Reply:
Ok, thanks!